docs/src/pages/components/button/demo.tsx

```diff
@@ -1,114 +1,166 @@
-import React from 'react';
-import { Button, ButtonSizes, ButtonVariations } from '@aws-amplify/ui-react';
-import { FieldLabeler } from '@/components/FieldLabeler';
-import { Example } from '@/components/Example';
+import * as React from 'react';
+
+import {
+  Button,
+  ButtonSizes,
+  ButtonVariations,
+  ButtonProps,
+  Flex,
+  SwitchField,
+  SelectField,
+  TextField,
+} from '@aws-amplify/ui-react';
+
+import { Demo } from '@/components/Demo';
+
+const propsToCode = (props) => {
+  return (
+    `<Button` +
+    (props.disabled ? `\n  isDisabled={${props.disabled}}` : '') +
+    (props.fullWidth ? `\n  isFullWidth={${props.fullWidth}}` : '') +
+    (props.loading ? `\n  isLoading={${props.loading}}` : '') +
+    (props.variation
+      ? `\n  variation=${JSON.stringify(props.variation)}`
+      : '') +
+    (props.size ? `\n  size=${JSON.stringify(props.size)}` : '') +
+    `
+  loadingText=${JSON.stringify(props.loadingText)}
+  onClick={() => alert('hello')}
+  ariaLabel=${JSON.stringify(props.ariaLabel)}
+>
+  Click me!
+</Button>`
+  );
+};
+
+const PropControls = (props) => {
+  return (
+    <Flex direction="column">
+      <SelectField
+        name="variation"
+        id="variation"
+        label="Variation"
+        value={props.variation}
+        onChange={(event) =>
+          props.setVariation(event.target.value as ButtonVariations)
+        }
+      >
+        <option value="">default</option>
+        <option value="primary">Primary</option>
+        <option value="link">Link</option>
+      </SelectField>
+
+      <SelectField
+        name="Size"
+        id="size"
+        label="Size"
+        value={props.size}
+        onChange={(event) => props.setSize(event.target.value as ButtonSizes)}
+      >
+        <option value="">default</option>
```

Comment:
Just caught this from the video, so not sure if we needed to Uppercase for consistency 

```suggestion
        <option value="">Default</option>
```